app/src/pages/admin/allUsersPage/allUsersPage.jsx

```diff
@@ -54,19 +121,98 @@ export class AllUsersPage extends Component {
     showModalAction: () => {},
     onChangePage: () => {},
     onChangePageSize: () => {},
     loading: false,
     users: [],
+    selectedUsers: [],
+    toggleAllUsersAction: () => {},
+    unselectAllUsersAction: () => {},
+    toggleUserSelectionAction: () => {},
+    userInfo: {},
+    deleteItemsAction: () => {},
+    showScreenLockAction: () => {},
+    hideScreenLockAction: () => {},
+    showNotification: () => {},
+    fetchAllUsersAction: () => {},
   };
 
   getBreadcrumbs = () => [
     {
-      title: this.props.intl.formatMessage(titleMessage.pageTitle),
+      title: this.props.intl.formatMessage(messages.pageTitle),
     },
   ];
-
+  get excludeFromSelection() {
+    const { userInfo } = this.props;
+    return [userInfo];
+  }
+  get selectedUsersNames() {
+    return this.props.selectedUsers.map((user) => `"${user.fullName}"`).join(', ');
+  }
+  get usersAvailableForSelection() {
+    const {
+      users,
+      userInfo: { id },
+    } = this.props;
+    return users.filter((user) => user.id !== id);
+  }
+  handleOneUserSelection = (value) => {
+    this.props.toggleUserSelectionAction(value);
+  };
+  handleToggleAllUserSelection = () => {
+    const users = this.usersAvailableForSelection;
+    this.props.toggleAllUsersAction(users);
+  };
+  confirmDeleteItems = (items) => {
+    const ids = items.map((item) => item.id).join(',');
+    this.props.showScreenLockAction();
+    fetch(URLS.usersBulkOperations(ids), {
+      method: 'delete',
+    })
+      .then(() => {
+        this.props.unselectAllUsersAction();
+        this.props.fetchAllUsersAction();
+        this.props.hideScreenLockAction();
+        this.props.showNotification({
+          message:
+            items.length === 1
+              ? this.props.intl.formatMessage(messages.success)
+              : this.props.intl.formatMessage(messages.successMultiple),
+          type: NOTIFICATION_TYPES.SUCCESS,
+        });
+      })
+      .catch(() => {
+        this.props.hideScreenLockAction();
+        this.props.showNotification({
+          message:
+            items.length === 1
+              ? this.props.intl.formatMessage(messages.error)
+              : this.props.intl.formatMessage(messages.errorMultiple),
+          type: NOTIFICATION_TYPES.ERROR,
+        });
+      });
+  };
+  handlerDelete = () => {
+    const { selectedUsers, intl, userId } = this.props;
+    this.props.deleteItemsAction(this.props.selectedUsers, {
+      onConfirm: this.confirmDeleteItems,
+      header:
+        selectedUsers.length === 1
+          ? intl.formatMessage(messages.deleteModalHeader)
+          : intl.formatMessage(messages.deleteModalMultipleHeader),
+      mainContent:
+        selectedUsers.length === 1
+          ? intl.formatMessage(messages.deleteModalContent, { name: this.selectedUsersNames })
+          : intl.formatMessage(messages.deleteModalMultipleContent, {
+              names: this.selectedUsersNames,
+            }),
+      userId,
```

Comment:
Is this field needed? Users do not have `owner` field, so we don't need to compare it.